app/src/main/java/com/github/neighbortrader/foodboardapp/clientmodel/User.java

```diff
@@ -103,6 +107,10 @@ public static User loadFromSharedPreferences() {
         return user;
     }
 
+    public boolean addOffer(Offer offerToAd) {
+        return this.offerList.add(offerToAd);
+    }
+
     public void deleteToken() {
         this.setJwtToken(null);
```

Comment:
Das im User der Token gelöscht wird finde ich strange, ist eigentlich nicht die Aufgabe des Users

Reply:
Wo sollte er deiner Meinung nach gelöscht werden?

Reply:
nen TokenHandler zumindest wird das in Angular so gemacht